test/installation-test/test-with-env-creds.js

```diff
@@ -0,0 +1,26 @@
+/**
+ *  Usage:
+ *
+ *  Assumes you have a .env with the following exported vars:
+ *   SIERRA_KEY=...
+ *   SIERRA_SECRET=...
+ *   SIERRA_URL=https://.../v6/
+ *
+ *  node ./test-with-env-creds
```

Comment:
This should actually be `source .env; node ./test-with-env-creds` - my bad. That should correctly load the contents of `.env` into `process.env` (assuming `.env` includes lines like `export SET SIERRA_KEY=...`)